src/icon-button.jsx

```diff
@@ -54,28 +54,83 @@ const IconButton = React.createClass({
   },
 
   propTypes: {
+    /**
+     * Can be used to pass a font icon as the icon for the button.
+     */
     children: React.PropTypes.node,
 
     /**
      * The css class name of the root element.
      */
     className: React.PropTypes.string,
+
+    /**
+     * Disables the icon button.
+     */
     disabled: React.PropTypes.bool,
```

Comment:
We don't have the default value. Can we add `disabled: false` to the `getDefaultProps`?